scripts/start.sh

```diff
@@ -14,14 +14,13 @@ sleep 5
 
 if [ $? -eq 0 ]; then
   echo "Checking for updates"
-  if [ $(git status -uno | grep -q "Your branch is up to date") ] ||
```

Comment:
When we were chatting yesterday I didn't see that `||`. It makes me think it should stay. Unless we'll never hit the condition where we have no changes and no build directory?

Reply:
oh right! I knew where the issue is... the directory check needs `../build`